en/developer_guide/media_standalone_display.rst

```diff
@@ -0,0 +1,186 @@
+Medias display
+==============
+
+Context
+-------
```

Comment:
I think I grasped the goal of this doc page. So I think you should start (before context) to say that there is a bundle that is available outside of the whole orchestra stack, standalone (apart for the BaseBundle I guess ?), etc. Some kind of intro so the reader understands that he can use this bundle in his sf project.